### PR TITLE
Extract method to copy list of lists

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/util/MoreLists.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+package com.facebook.presto.util;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static java.util.Objects.requireNonNull;
+
+public class MoreLists
```
Comment (kokosing) [inline presto-main/src/main/java/com/facebook/presto/util/MoreLists.java]: PrestoLists?

Comment (kokosing) [inline presto-main/src/main/java/com/facebook/presto/util/MoreLists.java]: PrestoColections?

File: presto-main/src/main/java/com/facebook/presto/sql/planner/plan/GroupIdNode.java [hunk 2]
```diff
@@ -60,7 +61,7 @@ public GroupIdNode(@JsonProperty("id") PlanNodeId id,
     {
         super(id);
         this.source = requireNonNull(source);
-        this.groupingSets = ImmutableList.copyOf(requireNonNull(groupingSets));
+        this.groupingSets = listOfListsCopy(requireNonNull(groupingSets, "groupingSets is null"));
```
Comment (kokosing) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/plan/GroupIdNode.java]: there is RNN already in the `listOfListCopy`

File: presto-main/src/main/java/com/facebook/presto/util/MoreLists.java (hunk 1 above)
Comment (kokosing) [inline presto-main/src/main/java/com/facebook/presto/util/MoreLists.java]: ImmutableCollectors?

File: presto-parser/src/main/java/com/facebook/presto/sql/tree/GroupingSets.java [hunk 3]
```diff
@@ -47,7 +48,7 @@ private GroupingSets(Optional<NodeLocation> location, List<List<QualifiedName>>
         super(location);
         requireNonNull(sets, "sets is null");
         checkArgument(!sets.isEmpty(), "grouping sets cannot be empty");
-        this.sets = ImmutableList.copyOf(sets.stream().map(ImmutableList::copyOf).collect(Collectors.toList()));
+        this.sets = sets.stream().map(ImmutableList::copyOf).collect(toImmutableList());
```
Comment (kokosing) [inline presto-parser/src/main/java/com/facebook/presto/sql/tree/GroupingSets.java]: if you would put into immutableCollectors then maybe you could use it here without calling map function

File: presto-main/src/main/java/com/facebook/presto/sql/planner/plan/GroupIdNode.java (hunk 2 above)
Comment (findepi) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/plan/GroupIdNode.java]: yes, but yields better exc msg

File: presto-main/src/main/java/com/facebook/presto/util/MoreLists.java (hunk 1 above)
Comment (findepi) [inline presto-main/src/main/java/com/facebook/presto/util/MoreLists.java]: Can be `PrestoLists`, but there is already `MoreMaps` in the same package. Please reconsider.

File: presto-parser/src/main/java/com/facebook/presto/sql/tree/GroupingSets.java (hunk 3 above)
Comment (findepi) [inline presto-parser/src/main/java/com/facebook/presto/sql/tree/GroupingSets.java]: I don't understand.

**Comment by findepi** [comment]
@kokosing comments answered

File: presto-main/src/main/java/com/facebook/presto/util/MoreLists.java (hunk 1 above)
Comment (kokosing) [inline presto-main/src/main/java/com/facebook/presto/util/MoreLists.java]: Ok, then lets stick with `MoreLists`, we could then rename them all together

File: presto-parser/src/main/java/com/facebook/presto/sql/tree/GroupingSets.java (hunk 3 above)
Comment (kokosing) [inline presto-parser/src/main/java/com/facebook/presto/sql/tree/GroupingSets.java]: nvm

**Pull request merged**